### ISSUE TITLE
[Feature request] Search feature

### ISSUE THREAD

**Issue by Regala** [dariusztytko/progress-burp#5]
Hi @dariusztytko , hope all is well. What do you think of having a 'search' feature, either in the main tab or another pane to facilitate searching for endpoints collected/covered?

Thanks!

**Comment by iamRjarpan** [comment]
Yeah, It allow to search based on tags, But it will be aswome if I can search everything
